access/resource_permissions_test.go

```diff
@@ -172,6 +175,44 @@ func TestResourcePermissionsDelete_error(t *testing.T) {
 	assert.Error(t, err)
 }
 
+func TestResourcePermissionsDelete_Job(t *testing.T) {
+	d, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:          http.MethodPut,
+				Resource:        "/api/2.0/preview/permissions/jobs/101",
+				ExpectedRequest: ObjectACL{},
+			},
+		},
+		Resource: ResourcePermissions(),
```

Comment:
can you add values in `HCL` property here? I see nowhere the `object_type = "job"` passed in here :)